websites/H/HBO Max/dist/metadata.json

```
@@ -15,7 +15,7 @@
     "play.hbomax.com",
     "help.hbomax.com"
   ],
-  "version": "1.0.5",
+  "version": "1.1.0",
```

I'd say bump to 2.0.0 as it is quite significant changes 
```suggestion
  "version": "2.0.0",
```